dissect/target/tools/query.py

```diff
@@ -373,23 +373,27 @@ def main() -> None:
                 continue
 
             rs = record_output(args.strings, args.json)
-            for entry in record_entries:
+            for func_generator in record_entries:
```

Comment:
`record_entries` -> `record` makes a lot of sense, but I don't necessarily find this one more readable.